adflow/pyADflow.py

```diff
@@ -1183,6 +1186,107 @@ def _solverunsteady(self, surfaceMeshCallback=None, volumeMeshCallback=None):
             self.solveTimeStep()
             self.writeSolution()
 
+    def getConvergenceHistory(self, workUnitTime=None):
```

Comment:
What use-cases did you have in mind with `getConvergenceHistory`? I realize this can be used for various things, but I mostly see this as a postprocessing thing where you would dump the dict to a file.

Reply:
I agree. 
I have added an example (from my run script) about how it could be used